frontend/views/components/Badge.vue

```diff
@@ -1,12 +1,20 @@
 <template lang="pug">
-span.c-badge(v-if='number') {{number}}
+span.c-badge(
+  :class='`is-${type}`'
+  :aria-label='`New ${$slots.default ? $slots.default[0].text : ""} notifications`'
```

Comment:
Since this is a user-facing string, it too needs to pass through the `L` function. So I would recommend something like (typing this up from memory):

```
:aria-label='L("{num} notifications", { num: $slots.default ? $slots.default[0].text : 0 })'
```

Please double-check also via the browser inspector tool that this text is in fact being set correctly on this attribute.

Reply:
Thanks greg. I will work on it soon. :)

Reply:
Thanks greg! I will make sure to merge it before I work on your change request, maybe tonight!